### PR TITLE
wrote stupid commands for climber

### PR DESCRIPTION
**Pull request by Gabriel-J-Young**

### CODE REVIEW

File: Competition2018/src/main/java/competition/subsystems/climb/commands/AscendClimberCommand.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	@Inject
+	public AscendClimberCommand (ClimbSubsystem climb) {
+		this.climb = climb;
+		this.requires(climb);
+	}
+	
+	@Override
+	public void initialize() {
+		log.info("Initializing");
+
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/climb/commands/AscendClimberCommand.java]: remove blank lines like these at the end of functions

File: Competition2018/src/main/java/competition/subsystems/climb/commands/DecendClimberCommand.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	@Inject
+	public DecendClimberCommand (ClimbSubsystem climb) {
+		this.climb = climb;
+		this.requires(climb);
+	}
+	
+	@Override
+	public void initialize() {
+		log.info("Initializing");
+
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/climb/commands/DecendClimberCommand.java]: remove blank line

File: Competition2018/src/main/java/competition/subsystems/climb/commands/StopClimberCommand.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	@Inject
+	public StopClimberCommand (ClimbSubsystem climb) {
+		this.climb = climb;
+		this.requires(climb);
+	}
+	
+	@Override
+	public void initialize() {
+		log.info("Initializing");
+
```
Comment (JohnGilb) [inline Competition2018/src/main/java/competition/subsystems/climb/commands/StopClimberCommand.java]: remove blank line

**Review by JohnGilb — CHANGES_REQUESTED** [review]

File: Competition2018/src/main/java/competition/subsystems/climb/commands/AscendClimberCommand.java (hunk 1 above)
Comment (Gabriel-J-Young) [reply Competition2018/src/main/java/competition/subsystems/climb/commands/AscendClimberCommand.java]: done

File: Competition2018/src/main/java/competition/subsystems/climb/commands/DecendClimberCommand.java (hunk 2 above)
Comment (Gabriel-J-Young) [reply Competition2018/src/main/java/competition/subsystems/climb/commands/DecendClimberCommand.java]: done

File: Competition2018/src/main/java/competition/subsystems/climb/commands/StopClimberCommand.java (hunk 3 above)
Comment (Gabriel-J-Young) [reply Competition2018/src/main/java/competition/subsystems/climb/commands/StopClimberCommand.java]: done

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**